.gitignore

```diff
@@ -8,16 +12,336 @@
 # testing
 /coverage
 
-# production
-/build
```

Comment:
The build folder should still be ignored as it is generated from running `npm run build` or `npm run deploy`

Reply:
done